app/views/categories/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render 'shared/header', title: "Categories", back_link: false %>
+<section class="container">
+<div class="total-header">
+    <% expense_count = @unique_expenses.size %>
+    <p>Spending: <%= number_to_currency(@total_expense) %></p>
```

Comment:
[OPTIONAL]
- The total amount of spending you added to the home page does not change when a new expense is added. 
![image](https://github.com/uchexm/Spendrit/assets/45460192/499fda2e-3056-44cc-8ba4-3ee44bc5a346)

Reply:
Done and done 😎